app/src/main/java/com/zulip/android/ZulipApp.java

```diff
@@ -147,6 +147,11 @@ public void setZulipActivity(ZulipActivity zulipActivity) {
     //on switch theme activity is recreated
     private boolean isThemeSwitchedFromHome = true;
 
+    //account for header
+    //will be helpful when them is switched
+    private int homeViewHeaderCount = 0;
```

Comment:
default value for integer member variables will be 0, hence we can omit this initialisation.